### PR TITLE
Misc improvements

### PR DESCRIPTION
**Pull request by BenCat07**
This just has a few miscelanous improvements and fixes to existing things.

- Rework Slow aim (Previous one had major issues due to the way it worked
- Fix the Critmeter Text not being properly alligned
- Improve The chat censor filter
- Fix desyncs with crithack

### CODE REVIEW

File: src/hacks/AntiAntiAim.cpp [hunk 1]
```diff
@@ -64,17 +64,64 @@ static float resolveAngleYaw(float angle, brutedata &brute)
...
+                CachedEntity *dot_ent = ENTITY(i);
+                // Not a sniper dot
+                if (CE_BAD(dot_ent) || dot_ent->m_iClassID() != CL_CLASS(CSniperDot))
+                    continue;
+                // Not the sniperdot of that entity
+                if (HandleToIDX(CE_INT(dot_ent, netvar.m_hOwnerEntity)) != ent->m_IDX)
+                    continue;
+                // Found sniper dot
+                sniper_dot = dot_ent;
+            }
```
Comment (TotallyNotElite) [inline src/hacks/AntiAntiAim.cpp:96]: What about doing this once per CM and storing that in an array? This seems very inefficient to me.

File: src/hooks/DispatchUserMessage.cpp [hunk 2]
```diff
@@ -215,7 +215,7 @@ DEFINE_HOOKED_METHOD(DispatchUserMessage, bool, void *this_, int type, bf_read &
                 break;
             }
 
-            std::vector<std::string> res = { "skid", "script", "cheat", "hak", "hac", "f1", "hax", "vac", "ban", "bot", "report", "kick" };
+            std::vector<std::string> res = { "skid", "script", "cheat", "hak", "hac", "f1", "hax", "vac", "ban", "bot", "report", "kick", "h@x", "hcak", "chaet", "one" };
```
Comment (TotallyNotElite) [inline src/hooks/DispatchUserMessage.cpp:218]: Why did you add h@x when the "@" is already replaced.

File: src/hacks/AntiAntiAim.cpp (hunk 1 above)
Comment (BenCat07) [reply src/hacks/AntiAntiAim.cpp:96]: Hm true i should store all sniper dots first, then use them

File: src/hooks/DispatchUserMessage.cpp (hunk 2 above)
Comment (BenCat07) [reply src/hooks/DispatchUserMessage.cpp:218]: Will fix
